<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rig_obj</class>
 <widget class="QDialog" name="rig_obj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rig Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="main_layout">
     <item>
      <layout class="QHBoxLayout" name="module_lay">
       <item>
        <layout class="QVBoxLayout" name="module_lab_lay">
         <item>
          <widget class="QLabel" name="module_type_lab">
           <property name="text">
            <string>Module Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="module_side_lab">
           <property name="text">
            <string>Module Side:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="module_set_lay">
         <item>
          <widget class="QComboBox" name="module_type_cb">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>propCmds</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>pistonCmds</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>wheelCmds</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="module_name_lay">
           <item>
            <widget class="QComboBox" name="module_side_cb">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>C</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>L</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>R</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="module_name_lab">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Module Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="module_name_led">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="main_gb">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="general_gb">
          <property name="title">
           <string>General Setup</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="attr_lay">
             <item>
              <layout class="QVBoxLayout" name="attr_lab_lay">
               <item>
                <widget class="QLabel" name="amount_lab">
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="statusTip">
                  <string/>
                 </property>
                 <property name="text">
                  <string>Amount:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="size_lab">
                 <property name="text">
                  <string>Size:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="shape_lab">
                 <property name="text">
                  <string>Shape:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="color_lab">
                 <property name="text">
                  <string>Color:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="buffer_lab">
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="statusTip">
                  <string/>
                 </property>
                 <property name="text">
                  <string>Buffer:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="attr_led_lay">
               <item>
                <widget class="QLineEdit" name="amount_led">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="size_led">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="shape_led">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="color_led">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="buffer_led">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="attr_sli_lay">
               <item>
                <widget class="QSlider" name="amount_sli">
                 <property name="minimumSize">
                  <size>
                   <width>300</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="size_sli">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="shape_sli">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="color_sli">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="buffer_sli">
                 <property name="minimumSize">
                  <size>
                   <width>300</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="general_lin">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="chb_lay">
             <item>
              <layout class="QVBoxLayout" name="chba_lay">
               <item>
                <widget class="QCheckBox" name="mirror_chb">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>Enable Mirror:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="control_chain_chb">
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>Control Chain:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="chba_spa">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="chbb_lay">
               <item>
                <widget class="QCheckBox" name="asymmetry_chb">
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>Enable Asymmetry:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="gimbal_control_chb">
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>Add Gimbal Control:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="chbb_spa">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="cb_lay">
               <item>
                <layout class="QHBoxLayout" name="mirror_axis_lay">
                 <item>
                  <widget class="QLabel" name="mirror_axis_lab">
                   <property name="text">
                    <string>Mirror Axis:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="mirror_axis_cb">
                   <property name="maximumSize">
                    <size>
                     <width>55</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <item>
                    <property name="text">
                     <string>x</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>y</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>z</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="rotate_order_lay">
                 <item>
                  <widget class="QLabel" name="rotate_order_lab">
                   <property name="text">
                    <string>Rotate Order:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="rotate_order_cb">
                   <property name="minimumSize">
                    <size>
                     <width>55</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>55</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <item>
                    <property name="text">
                     <string>xyz</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>yzx</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>zxy</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>xzy</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>yxz</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>zyx</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="joint_gb">
          <property name="title">
           <string>Joint Setup</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="joint_chb_lay">
             <item>
              <widget class="QCheckBox" name="use_joints_chb">
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="text">
                <string>Use Joints:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="joint_chba_spa">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="create_ik_chb">
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="text">
                <string>Create IK Handle:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="joint_chbb_spa">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="ik_solver_lab">
               <property name="text">
                <string>IK Solver:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="ik_solver_cb">
               <item>
                <property name="text">
                 <string>ikSCsolver</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="joint_lin">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="ik_lay">
             <item>
              <layout class="QVBoxLayout" name="ik_lab_lay">
               <item>
                <widget class="QLabel" name="start_joint_lab">
                 <property name="text">
                  <string>Start Joint:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="end_effector_lab">
                 <property name="text">
                  <string>End Effector:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="parent_control_lab">
                 <property name="text">
                  <string>Parent Control:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="ik_cb_lay">
               <item>
                <widget class="QComboBox" name="start_joint_cb">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="end_effector_cb">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="parent_control_cb">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="create_spa">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="create_btn">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
